alfred/Makefile

```diff
@@ -8,10 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=alfred
-PKG_VERSION:=2018.1
-PKG_RELEASE:=1
-PKG_HASH:=808fa6acf65c7a8e26405115176a5587157f746108cbe5dd974788eb05416d76
-
+PKG_VERSION:=2018.2
+PKG_RELEASE:=0
+PKG_HASH:=2bbe9c4fc19a6529c40f7c872406fad8070d145ede970806bd2194e07fba8018
```

Comment:
Maybe squash this into the first patch and add the enable-if-clause ...

Reply:
done https://github.com/openwrt-routing/packages/pull/390/files#diff-9cac72d591240485e6d7dd48f4aaa90eR101